GVRf/Framework/jni/objects/mesh.cpp

```diff
@@ -145,6 +146,66 @@ const float* Mesh::getBoundingBoxInfo() {
     return bounding_box_info_;
 }
 
+float* Mesh::getTransformedBoundingBoxInfo(glm::mat4 *Mat) {
+
+    if (have_bounding_box_ == false)
+        getBoundingBoxInfo();
+
+    glm::mat4 M = *Mat;
+    float a, b;
+    float *transformed_bounding_box = new float[6];
+
```

Comment:
Alright shall change the function signature to match the above. However it basically means a method call with side effects - something as a personal preference I do not like to have in my code.